cloudwash/entities/resources/discs.py

```diff
@@ -0,0 +1,51 @@
+from cloudwash.config import settings
+from cloudwash.entities.resources.base import DiscsCleanup
+from cloudwash.logger import logger
+from cloudwash.utils import dry_data
+
+
+class CleanDiscs(DiscsCleanup):
+    def __init__(self, client):
+        self.client = client
+        self._delete = []
+        self.list()
```

Comment:
not sure how this is working ? is that intentional ?

Reply:
Yes it is intentional. As soon as the class is initiated the listing should happen and all dry data must be updated for viewing without calling the explicit `cleanup()` method.

Reply:
This also helps in if user just wanted to list / stop or any other method without actually cleaning it or calling that function !